index.html

```diff
@@ -215,28 +215,31 @@ <h3>Skills</h3>
                 </li>
             </ul>
         </div>
-    </section>
-    <footer>
-        <section class="contact-me">
-            <h1 class="contact-me-text">contact me</h1>
-            <p class="contact-me-brief">If you have an application you are intrested in developing, a feature that
+    </section> 
+    <section class="contact-me">
+            <h1 class="contact-me-text">Contact me</h1>
+            <p class="contact-me-brief">
+                If you have an application you are intrested in developing, a feature that
                 you need built or a project that needs coding.
                 I'd love to help with it.
             </p>
-            <form action="">
+            <form 
+                action="https://formspree.io/f/xeqnbaww"
+                method="POST">
                 <div class="input-field-box">
-                    <input class="input-field" type="text" name="NAME" maxlength="30" value="Type your name">
-                    <input class="input-field" type="email" name="EMAIL" value="username@domain.ext">
-                    <textarea class="write-message" name="write message" placeholder="Write your message here"
-                        maximumlength="100" rows="5" cols="30">
-                    </textarea>
+                    <input class="input-field" type="text" name="name" maxlength="30" value="Type your name">
+                    <input class="input-field" type="email" name="email" value="Username@domain.ext">
+                    <textarea 
```

Comment:
- [ ] OPTIONAL: Instead of assigning value to your inputs, it would be better if you use a `placeholder` for a better user experience and accessiblity